src/Plugin/GraphQL/DataProducer/Entity/EntityLabel.php

```diff
@@ -22,6 +22,11 @@
  *     "entity" = @ContextDefinition("entity",
  *       label = @Translation("Entity")
  *     ),
+ *     "access" = @ContextDefinition("boolean",
```

Comment:
we should introduce new parameters at the end to be more compatible.

We should call this new parameter context "check_access" to be more clear. Yes, we use a different name on other dataproducers, but I think it still makes sense to use this better name.

Reply:
Ok, I updated the name of it.